src/components/Barchart.js

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from "react";
+import {
+  BarChart,
+  Bar,
+  Cell,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer
+} from "recharts";
+
+const data = [
+  {
+    date: "14/11/2019",
+    food: 1050,
+    cloth: 500,
+    medical: 500,
+    amt: 2050
+  },
+  {
+    date: "13/11/2019",
+    food: 2000,
+    cloth: 1000,
+    medical: 100,
+    amt: 3100
+  },
+  {
+    date: "11/11/2019",
+    food: 1950,
+    cloth: 500,
+    medical: 1400,
+    amt: 3850
+  }
+];
+
+export default class Example extends PureComponent {
```

Comment:
It seems it could be a dumb component. You could use a simple `export default function Cart() {}` here.